### PR TITLE
2800_Verify_executable_generator

### LINKED ISSUE #2800: Verify executable generator
For the next wargame we're going to need to be able to deploy Serge as an executable.

We should verify the executable generator runs properly (`yarn pkg-build`), and in particular check the MS-Windows executable works.

Note: it currently runs to completion, but the `macos` executable certainly fails:
```
./executable-macos 
running client with local server
(You can give the second parameter as the remote server URL)
/snapshot/serge/server/bin.js:55
    throw err
    ^

Error: Directory '/**/serge/data' was not included into executable at compilation stage. Please recompile adding it as asset or script.
    at error_ENOENT (pkg/prelude/bootstrap.js:657:19)
    at readdirFromSnapshot (pkg/prelude/bootstrap.js:1282:18)
    at Object.readdir (pkg/prelude/bootstrap.js:1349:5)
    at Object.<anonymous> (/snapshot/serge/server/bin.js:53:4)
    at Module._compile (pkg/prelude/bootstrap.js:1926:22)
    at Module._extensions..js (node:internal/modules/cjs/loader:1166:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:834:12)
    at Function.runMain (pkg/prelude/bootstrap.js:1979:12)
    at node:internal/main/run_main_module:17:47 {
  errno: -2,
  code: 'ENOENT',
  path: '/snapshot/serge/data',
  pkg: true
}

Node.js v18.5.0
```

### PR DESCRIPTION
**Pull request by lilitkarapetyan**
## 🧰 Issue

Fixes #2800

## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

**Comment by lilitkarapetyan** [comment]
Hello, **Note:**  I'm currently running a Windows operating system on my computer. I've managed to download the node_sqlite3.node file tailored for the Windows 64-bit (win64) platform successfully. I've also attempted to install it on Linux and macOS. If it doesn't work, can you  try downloading the corresponding file for Linux and macOS, using Node version 18.15.0.

**Comment by lilitkarapetyan** [comment]
Fixes #2800

**Comment by IanMayo** [comment]
Hello Lilit - we certainly aren't using those large json files in Serge anymore. You have removed the code that used them.

**Comment by IanMayo** [comment]
Hello @lilitkarapetyan :-)

In our current test implementation we're getting map tiles from the Internet.  But, it's almost certain that future wargames will require locally hosted tiles - so we should keep the `/tiles` route, please.

File: server/server.js [hunk 1]
```diff
@@ -117,26 +116,6 @@ const runServer = (
...
 
-  app.get('/cells/:filename', (req, res) => {
-    if (dataDir) {
-      return res.sendFile(
-        path.join(process.cwd(), dataDir, req.params.filename)
-      )
-    }
-    res.sendFile(path.join(__dirname, '../', 'data', req.params.filename))
-  })
-
```
Comment (IanMayo) [inline server/server.js:128]: Hello @lilitkarapetyan :-)

In our current test implementation we're getting map tiles from the Internet. But, it's almost certain that future wargames will require locally hosted tiles - so we should keep the /tiles route, please.

**Review by IanMayo — APPROVED** [review]

**Pull request merged**